packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -778,28 +778,20 @@ export class RemoteFluidDataStoreContext extends FluidDataStoreContext {
         this.baseGCDetailsP = new LazyPromise<IGarbageCollectionDetailsBase>(async () => {
             return (await props.getBaseGCDetails()) ?? {};
         });
+
+        if (props.snapshotTree !== undefined) {
+            this.summarizerNode.updateBaseSummaryState(props.snapshotTree);
+        }
     }
 
     private readonly initialSnapshotDetailsP = new LazyPromise<ISnapshotDetails>(async () => {
-        let tree: ISnapshotTree | undefined;
+        let tree = this.initSnapshotValue;
         let isRootDataStore = true;
 
-        if (typeof this.initSnapshotValue === "string") {
```

Comment:
@anthony-murphy Not sure if you are familiar with this code but I removed it since the call sites that create remote context only pass in ISnapshotTree | undefined.